src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheck.java

```diff
@@ -34,8 +34,7 @@
 import com.puppycrawl.tools.checkstyle.utils.TokenUtil;
 
 /**
- * Checks that particular class are never used as types in variable
- * declarations, return values or parameters.
+ * Checks that particular classes or interfaces are never used.
```

Comment:
Since class javadoc was changed dramatically, http://checkstyle.sourceforge.net/checks.html has to be changed too.

Reply:
done